src/Components/Blog.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react'
+import styles from '../css/Blog.module.css'
+import { NavBar } from "./Landing"
+import firebase from "../firebase/api"
+
+interface IMyComponentProps {
+    someDefaultValue: string
+}
+
+interface IMyComponentState {
+    emails: Array<String>
+}
+
+class Blog extends Component<IMyComponentProps, IMyComponentState> {
+    constructor(props: IMyComponentProps) {
+        super(props) 
+        this.state = {emails: []}
+    }
+
+    componentDidMount() {
```

Comment:
would prefer React Hooks and functional components: https://reactjs.org/docs/hooks-intro.html

Reply:
I see, I'm a little more used to class components but we needed it done in a pretty short time so I used what I was familiar with at the time but can def port over pretty soon :)

Reply:
ya this isn't blocking, feel free to move on. Just good to learn about now since it's been ~2 years since Hooks was announced and lifecycle methods may be deprecated soon.